docker-run.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+set
```

Comment:
Is this intentionally included?

Reply:
when running the script I've tons of output so I would say it's more for debugging purpose and hasn't been removed ?

Reply:
Exact, just a rest from `set -x`